library/node

```diff
@@ -3,63 +3,63 @@
 Maintainers: The Node.js Docker Team <https://github.com/nodejs/docker-node> (@nodejs)
 GitRepo: https://github.com/nodejs/docker-node.git
 
-Tags: 7.9.0, 7.9, 7, latest
-GitCommit: e1103db1e7330f620ec4b5961b93936da11becdf
-Directory: 7.9
+Tags: 7.10.0, 7.10
```

Comment:
Looks like `latest` tag got lost; which also means `7-*`, `onbuild`, `slim`, and `wheezy` are missing too.